desc/protoparse/ast.go

```diff
@@ -785,18 +785,20 @@ type enumValueNode struct {
 	basicCompositeNode
 	name    *identNode
 	options []*optionNode
-	// only one of these two will be set
-	number  *intLiteralNode
-	numberN *negativeIntLiteralNode
+
+	// only one of these two will be set:
+
+	numberP *intLiteralNode         // positive numeric value
```

Comment:
So any future changes referencing the field will more likely notice that this is only for positive values -- code either needs explicit check to decide whether to use `numberP` or `numberN` or it should just use the `getNumber()` accessor.